docs/api-reference/data-fetching/getStaticPaths.md

```diff
@@ -0,0 +1,200 @@
+---
+description: Fetch data and generate static pages with `getStaticProps`.
+---
+
+# `getStaticPaths`
+
+<details>
+  <summary><b>Version History</b></summary>
+
+| Version  | Changes                                                                                                         |
+| -------- | --------------------------------------------------------------------------------------------------------------- |
+| `v9.5.0` | Stable [Incremental Static Regeneration](/docs/basic-features/data-fetching/incremental-static-regeneration.md) |
```

Comment:
Follow up PR - let's add more updates here, we've definitely added more stuff between here and 9.5

Reply:
https://github.com/vercel/next.js/issues/31953